packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -518,7 +566,7 @@ const styles = StyleSheet.create({
     textAlign: 'center',
   },
   tableCellTitleNumber: {
-    marginLeft: 15 + 20 + 20,
```

Comment:
nit: im guessing that your adding values together as they signify something, consider using a constant so its clear what 20 means and what other places are 20 for the same reason

Reply:
@Pedro-vk